packages/ffe-cards-react/src/IconCard/IconCard.md

```diff
@@ -0,0 +1,49 @@
+Ikonkort er kort med en gitt maksbredde og et venstrestilt ikon.
+Den tilbyr alle underkomponenter (CardName, Title, Subtext og Text) og plasserer dem i
+gitt rekkefølge nedover i kortet, med teksten venstrestilt, til høyere for ikonet.
+
+```jsx
+const { GrafOppIkon } = require('../../../ffe-icons-react');
+
+<React.Fragment>
+    <IconCard icon={<GrafOppIkon />}>
+        {({ CardName, Title, Subtext, Text }) => (
+            <React.Fragment>
+                <CardName>Kortnavn</CardName>
+                <Title>Tittel</Title>
+                <Subtext>En liten undertekst</Subtext>
+                <Text>Her kan man ha tekst</Text>
+            </React.Fragment>
+        )}
+    </IconCard>
+</React.Fragment>;
+```
```

Comment:
Her kan du vel droppe Fragment rundt `<IconCard />`

Reply:
Ta en runde på alle eksemplene, var et par steder til det kunne blitt fjerna :)

Reply:
Yeah, I saw there was one more in TextCard.md. Fixed locally.